### PR TITLE
Fixes #19136: Delay remote trigger on change

### PR DESCRIPTION
**Pull request by peckpeck**
https://issues.rudder.io/issues/19136

### CODE REVIEW

**Review by amousset — APPROVED** [review]

**Review by amousset — COMMENTED** [review]
missing fi

File: webapp/sources/rudder/rudder-core/src/main/resources/hooks.d/policy-generation-finished/60-trigger-node-update [hunk 1]
```diff
@@ -61,6 +64,20 @@ fi
 
 # Signal to cf-serverd that it shall look-up the new promises
 if [ -n "${RUDDER_NODE_IDS[0]}" ] && (( ${TO_UPDATE} > 0 )) ; then
+  # wait for the server to be reloaded if needed
+  WAIT="yes"
+  if [ -f "${ACL_BACK}"  ]
+  then
```
Comment (amousset) [inline webapp/sources/rudder/rudder-core/src/main/resources/hooks.d/policy-generation-finished/60-trigger-node-update:70]: missing fi for this condition

**Comment by peckpeck** [comment]
PR updated with a new commit

**Review by amousset — APPROVED** [review]

**Comment by peckpeck** [comment]
OK, squash merging this PR

**Comment by peckpeck** [comment]
OK, merging this PR

**Pull request merged**